servicemesher/client/pom.xml

```diff
@@ -275,91 +264,23 @@
 		</dependency>
 	</dependencies>
 
-
-
 	<build>
-		<defaultGoal>install</defaultGoal>
-		<directory>target</directory>
-		<finalName>${project.artifactId}-${project.version}</finalName>
 		<plugins>
-			<!-- Jacoco Code Coverage -->
 			<plugin>
-				<groupId>org.jacoco</groupId>
-				<artifactId>jacoco-maven-plugin</artifactId>
-				<version>0.8.1</version>
-				<executions>
-					<execution>
-						<configuration>
-							<destFile>${sonar.jacoco.reportPath}</destFile>
-						</configuration>
-						<id>pre-test</id>
-						<phase>process-test-classes</phase>
-						<goals>
-							<goal>prepare-agent</goal>
-						</goals>
-					</execution>
-				</executions>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.0</version>
+				<configuration>
+					<source>8</source>
```

Comment:
Should this be changed to <release>11</release>

Reply:
changed pom to be consistent with other examples

Reply:
Yes. If you are using 2.0.2-SNAPSHOT version